openedxstats/apps/sites/apps.py

```diff
@@ -1,6 +1,8 @@
+import django
 from django.apps import AppConfig
 
 
 class SitesConfig(AppConfig):
     name = 'sites'
-
+    if django.VERSION >= (3, 2):
```

Comment:
https://docs.djangoproject.com/en/3.2/ref/applications/#django.apps.AppConfig.default

prevent Django from selecting a configuration class automatically.
it already has defined path in Installed apps.

Reply:
https://github.com/edx/openedxstats/blob/master/openedxstats/settings/base.py#L44